utils/ceramicAuth.tsx

```
@@ -72,43 +66,65 @@ const authenticateEthPKH = async (
-          resources: compose.resources,
-        });
-        // Set the session in localStorage.
-        localStorage.setItem('ceramic:eth_did', session.serialize());
-      }
 
-      // Set our Ceramic DID to be our session DID.
-      compose.setDID(session.did);
-      ceramic.did = session.did;
-      localStorage.setItem('display did', session.did.toString());
-      resolve(true);
+        if (session && session.did) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)